File: scripts/playground/patch_db.py
import psycopg2
conn = psycopg2.connect("postgresql://yayanews:Jia1009po@127.0.0.1:5432/yayanews")
cur = conn.cursor()
cur.execute("UPDATE topics SET cover_image = REPLACE(cover_image, '.png', '.jpg')")
conn.commit()
print("Updated rows:", cur.rowcount)
cur.close()
conn.close()

File: scripts/playground/patch_db.js
const { Client } = require('pg');

const client = new Client('postgresql://yayanews:Jia1009po@127.0.0.1:5432/yayanews');

async function run() {
  try {
    await client.connect();
    const res = await client.query("UPDATE topics SET cover_image = REPLACE(cover_image, '.png', '.jpg')");
    console.log(`Updated ${res.rowCount} rows`);
  } catch(e) {
    console.error(e);
  } finally {
    await client.end();
  }
}

run();
